sme_finance_application_schema/finance_application_v3

```diff
@@ -17,6 +17,7 @@
             "items": {
                 "type": "object",
                 "properties": {
+                    "value_of_property": "integer",
```

Comment:
Needs a description - I can see we don't want to tie this to a particular property, but in that case it's more about assets that the Director is willing/able to use to guarantee the loan rather than the value of a particular property.